yt/frontends/enzo/simulation_handling.py

```diff
@@ -477,7 +477,7 @@ def _calculate_cycle_outputs(self):
             or "CycleSkipDataDump" not in self.parameters
             or self.parameters["CycleSkipDataDump"] <= 0.0
         ):
-            return []
+            return
```

Comment:
this case is a bit weird. The function has only one explicitly return statement so it will usually return `None` except here where it return an empty list. I figure it is probably useless now but the function might need to be fixed differently

Reply:
Probably need @brittonsmith 's eyes on this